### DISCUSSION TITLE
How to create recurring transactions for salary?

### DISCUSSION THREAD

**Discussion by JClishe** [firefly-iii/firefly-iii#8253] [Firefly III]
New user here. I've been using Banktivity on my Mac for many years and am now trying to determine whether selfhosting Firefly will suit my needs so I can avoid Banktivity's annual subscription fees (as well as not relying on someone else to secure my data).

I'm trying to wrap my head around how to configure recurring transactions for my paycheck. My paycheck direct deposits into several accounts, and the amount for each account is not the same from check to check (but close). Currently in Banktivity there's a view for all upcoming recurring transactions. In it I select the deposit for each account individually, adjust the deposit amount, then post the transaction. I also sometimes do this several days to a week prior to my pay date.

So that's what I'm trying to recreate. Do I need to create a recurring transaction for the deposit into each account, or is there a way to create a single transaction with splits that deposits into each account? Is there a simple way to adjust the deposit amount prior to posting?  I'm also confused by needing to define a source account for a paycheck deposit. Is it my employer?

**Comment by JC5** [comment]
1. You can't add dynamic amounts to recurring transactions.
2. You can't create splits.
